mwana/apps/export/views.py

```diff
@@ -0,0 +1,132 @@
+# vim: ai ts=4 sts=4 et sw=4
+# Create your views here.
+import psycopg2
+from mwana import localsettings as settings
+from django.http import HttpResponse
+import json
+from mwana.apps.locations.models import Location
+from collections import OrderedDict
+
+
+def get_turnaround(request):
+    '''
+    key_value = ""
+    if ((request.method == 'GET') and ('key_value' in request.GET)):
+        key_value = request.GET['key_value']
+    '''
+    connection = psycopg2.connect(host=settings.DATABASES['default']['HOST'], database=settings.DATABASES['default']['NAME'],\
```

Comment:
Any reason for not using connection from django.db, i.e.:

from django.db import connection
cursor = connection.cursor()
...